client/src/remote/Plugins.js

```diff
@@ -88,6 +91,10 @@ export default class Plugins {
       createTab
     };
 
+    // Carbon React exports for the client plugins
+    global.carbon = CarbonReactExports;
```

Comment:
As indicated [here](https://github.com/camunda/camunda-modeler-plugin-helpers/pull/31#discussion_r2169824104), let's align with existing `vendor` exports, and not pollute the global scope more than necessary (due to legacy).
```suggestion
    global.vendor.carbon = CarbonReactExports;
```